src/quo2/core.cljs

```diff
@@ -49,10 +49,10 @@
     quo2.components.links.url-preview-list.view
     quo2.components.links.url-preview.view
     quo2.components.list-items.channel
-    quo2.components.list-items.community.view
     quo2.components.list-items.menu-item
     quo2.components.list-items.preview-list
     quo2.components.list-items.user-list
+    quo2.components.list-items.community.view
```

Comment:
why did you move this?  it was alpha sorted before?

Reply:
Sorry for that. I think it happens accidently while rebase.